src/devices/ServoMotor/ServoMotor.cs

```diff
@@ -0,0 +1,133 @@
+    {
+        private PwmChannel _pwmChannel;
+        private readonly int _maximumAngle;
+        private readonly int _maximumPulseWidthMicroseconds;
+        private readonly int _minimumPulseWidthMicroseconds;
+        private readonly double _servoAngleSlope;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServoMotor"/> class.
+        /// </summary>
+        /// <param name="maximumAngle">The maximum angle the servo motor can move.</param>
+        /// <param name="minimumPulseWidthMicroseconds">The minimum pulse width, in microseconds, that represent an angle for 0 degrees.</param>
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
+        private ServoMotor(int maximumAngle, int minimumPulseWidthMicroseconds, int maximumPulseWidthMicroseconds)
+        {
+            _maximumAngle = maximumAngle;
+            _minimumPulseWidthMicroseconds = minimumPulseWidthMicroseconds;
+            _maximumPulseWidthMicroseconds = maximumPulseWidthMicroseconds;
+            _servoAngleSlope = CalculateServoAngleSlope(maximumAngle, minimumPulseWidthMicroseconds, maximumPulseWidthMicroseconds);
+        }
+
+        /// <summary>
+        ///  Initializes a new instance of the <see cref="ServoMotor"/> class.
+        /// </summary>
+        /// <param name="pwmChannel">The PWM channel used to control the servo motor.</param>
+        /// <param name="maximumAngle">The maximum angle the servo motor can move.</param>
+        /// <param name="minimumPulseWidthMicroseconds">The minimum pulse width, in microseconds, that represent an angle for 0 degrees.</param>
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
+        public ServoMotor(
+            PwmChannel pwmChannel,
+            int maximumAngle = 180,
+            int minimumPulseWidthMicroseconds = 1_000,
+            int maximumPulseWidthMicroseconds = 2_000)
+            : this(maximumAngle, minimumPulseWidthMicroseconds, maximumPulseWidthMicroseconds)
+        {
+            _pwmChannel = pwmChannel;
+        }
+
+        /// <summary>
+        ///  Initializes a new instance of the <see cref="ServoMotor"/> class.
+        /// </summary>
+        /// <param name="chip">The PWM chip number used to control the servo motor.</param>
+        /// <param name="channel">The PWM channel number used to control the servo motor.</param>
+        /// <param name="maximumAngle">The maximum angle the servo motor can move.</param>
+        /// <param name="minimumPulseWidthMicroseconds">The minimum pulse width, in microseconds, that represent an angle for 0 degrees.</param>
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
+        public ServoMotor(
```

Comment:
I'd drop this overload - we don't have similar convenience methods for SPI/I2C devices (i.e. we take I2cDevice directly and not I2cSettings)

Reply:
In next commit